views/default/elgg.js.php

```diff
@@ -13,8 +13,10 @@
 
 JS;
 
-// For backwards compatibility...
-echo elgg_view('sprintf.js');
+?>
+//<script>
+require(['sprintf']);
```

Comment:
i think so too... it works, but i think i got lucky... would be nice to loose all custom libraries, so languages would just depend on sprintf